6_drivers_fetch.yml

```diff
@@ -67,7 +67,9 @@ targets:
     command: loop_tasks(task_plan = nldas_append_cube_plan, task_makefile = "6_fetch_append_cubes_tasks.yml")
 
   new_feather_plan:
-    command: create_cell_task_plan(nldas_new_cells, replete_time_range, nldas_new_cube_files, feather_dir = I('6_drivers/out/feather'), ind_dir=I('6_drivers/log/feather'))
+    command: create_cell_task_plan(nldas_new_cells, replete_time_range, nldas_new_cube_files,
```

Comment:
oops. Noting here that `append_cubes`, `new_cubes`, and `new_feathers` should be indicator files, not objects. Not sure if changing them now would trigger a rebuild of all of the files...?

Reply:
This comment is out of date with respect to new_feathers, right?

I don't think the files would get rebuilt, but I do think remake would re-check all the files represented by each of those .ind files.